src/main/java/com/eternalcode/discordapp/database/model/User.java

```diff
@@ -0,0 +1,34 @@
+package com.eternalcode.discordapp.database.model;
+
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+
+@DatabaseTable(tableName = "users")
```

Comment:
proponuje dodać jakiś prefix

Reply:
Nie widzę sensu prefixów kiedy to będzie w jednej bazie

Reply:
prefixy dodaję się po to by to nie kolidowało z innymi bazami, nie ważne czy będziesz robić to w jednej bazie czy nie

Reply:
Ale tutaj nie ma co kolidować, bot używa swojej osobnej bazy, i nie będzie to integrowane z innymi bazami więc nie ma co kolidować